codegen/src/api/errors.rs

```diff
@@ -20,8 +19,8 @@ pub fn generate_error_type_alias(
         return Ok(quote!());
     };
 
-    let error_type = type_gen.resolve_type_path(error_ty);
-    let error_ty = type_gen.resolve_type(error_ty);
+    let error_type = type_gen.resolve_type_path(error_ty)?;
+    let error_ty = type_gen.resolve_type(error_ty)?;
     let docs = &error_ty.docs;
     let docs = should_gen_docs
```

Comment:
Could this be infered from the `type_gen.settings()`?

Reply:
Oh, totally! Good catch!